cpp/tests/io/read_shapefile_polygon_test.cu

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <time.h>
+#include <sys/time.h>
+#include <string>
+
+#include <gtest/gtest.h>
+#include <utilities/error_utils.hpp>
+#include <tests/utilities/cudf_test_utils.cuh>
+#include <tests/utilities/cudf_test_fixtures.h>
+#include <cuspatial/shp_readers.hpp>
+#include <utility/utility.hpp> 
+
+struct ReadShapefilePolygonTest : public GdfTest 
+{
+};   
+
```

Comment:
Only one test?

Please test corner cases, such as empty files, degenerate files, files that aren't empty but have zero for some attributes, etc.

Reply:
More tests will be added after we figure out where to put data files. Currently, tests module is  disabled completely anyway.

Reply:
All you need is a test that reads a (trivial) invalid file and tests that the error reported is correct.

Reply:
Again, where should these test files be put? I thought we decided to table the issue when a suitable place can be found. Please see my comments on the slack channel.